requirements/required.txt

```diff
@@ -0,0 +1,10 @@
+efficientnet-pytorch==0.7.1
```

Comment:
why pin the required? Shouldn't keep it and install the latest version automatically?

Reply:
This file is only used for CI and is controlled by dependabot. The dependencies used during installation are found in pyproject.toml.